File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/get-vetting-batches_result.ts


import { IVettingBatch_Item } from './ivetting-batch_item';
import { IGetVettingBatches_Result } from './iget-vetting-batches_result';

export class GetVettingBatches_Result implements IGetVettingBatches_Result {
  
	public Batches?: IVettingBatch_Item[];
  
	public ErrorMessages: string[] = null;
}

File: GTTS/INL.VettingService.Models/GetVettingBatches_Result.cs
using INL.Services.Models;
using System.Collections.Generic;

namespace INL.VettingService.Models
{
    public class GetVettingBatches_Result : BaseResult, IGetVettingBatches_Result
    {
        public List<IVettingBatch_Item> Batches { get; set; }
    }
}
